src/renderer/models/beaconNode.ts

```diff
@@ -0,0 +1,11 @@
+export interface IBeaconNode {
+    url: string;
+}
+
+export class BeaconNode implements IBeaconNode {
+    public url: string;
```

Comment:
I think this should also hold info if it's local node like docker id or something (think about initializing dockers on startup and getting right docker from registry)

Reply:
Right, good input!